### PR TITLE
fix(ssl): ssl label upgrade max len

### PR DESCRIPTION
**Pull request by Fabriceli**
### Description
Upgrade SSL label len 64 to 256

Fixes # (issue)
https://github.com/apache/apisix-ingress-controller/issues/1768

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [x] I have explained the changes or the new features added to this PR
- [x] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: t/admin/ssl4.t [hunk 1]
```diff
@@ -364,3 +364,40 @@ location /t {
...
+--- config
+location /t {
+    content_by_lua_block {
+        local core = require("apisix.core")
+        local t = require("lib.test_admin")
+
+        local ssl_cert = t.read_file("t/certs/apisix.crt")
+        local ssl_key =  t.read_file("t/certs/apisix.key")
+        local data = {cert = ssl_cert, key = ssl_key, sni = "www.test.com",
+         labels = {"secret-name": "js-design-test-bigdata-data-app-service-router-my-secret-number-123456"}}
```
Comment (monkeyDluffy6017) [inline t/admin/ssl4.t:384]: extra space

Comment (monkeyDluffy6017) [inline t/admin/ssl4.t:384]: What does this line for?

Comment (monkeyDluffy6017) [inline t/admin/ssl4.t:384]: the parameters of the second line of functions are to the right of the left parenthesis of the first line.
```
return limit_conn_new("plugin-limit-conn", conf.conn, conf.burst,
                      conf.default_conn_delay)

```

**Comment by Fabriceli** [comment]
could you return the CI? it seems that the unit test is not stable @monkeyDluffy6017

**Comment by Fabriceli** [comment]
No error in my local for Code Lint CI:
```sh
$ make lint
[ info ] lint -> [ Start ]
./utils/check-lua-code-style.sh
+ luacheck -q apisix t/lib
Total: 0 warnings / 0 errors in 251 files
+ find apisix -name '*.lua' '!' -wholename apisix/cli/ngx_tpl.lua -exec ./utils/lj-releng '{}' +
+ grep -E 'ERROR.*.lua:' /tmp/check.log
+ true
+ '[' -s /tmp/error.log ']'
./utils/check-test-code-style.sh
+ find t -name '*.t' -exec grep -E '\-\-\-\s+(SKIP|ONLY|LAST|FIRST)$' '{}' +
+ true
+ '[' -s /tmp/error.log ']'
+ find t -name '*.t' -exec ./utils/reindex '{}' +
+ grep done. /tmp/check.log
+ true
+ '[' -s /tmp/error.log ']'
[ info ] lint -> [ Done ]
```

File: t/admin/ssl4.t [hunk 2]
```diff
@@ -364,3 +364,42 @@ location /t {
         t.test('/apisix/admin/ssls/1', ngx.HTTP_DELETE)
     }
 }
+
+
```
Comment (monkeyDluffy6017) [inline t/admin/ssl4.t:368]: Three blanks are needed

Comment (Fabriceli) [reply t/admin/ssl4.t:368]: done

**Comment by monkeyDluffy6017** [comment]
@Fabriceli Please fix the ci

**Comment by Fabriceli** [comment]
> @Fabriceli Please fix the ci

done

File: t/admin/ssl4.t [hunk 3]
```diff
@@ -16,7 +16,7 @@
 #
 use t::APISIX 'no_plan';
 
-log_level('debug');
+log_level('info');
```
Comment (monkeyDluffy6017) [inline t/admin/ssl4.t:19]: Why do you modify this?

Comment (Fabriceli) [reply t/admin/ssl4.t]: modified by mistake.

**Review by soulbird — APPROVED** [review]

**Review by monkeyDluffy6017 — APPROVED** [review]

**Pull request merged**